Homework/Develop/index.html

```diff
@@ -22,61 +23,69 @@ <h1>Hori<span class="seo">seo</span>n</h1>
                     <a href="#social-media-marketing">Social Media Marketing</a>
                 </li>
             </ul>
-        </div>
+
+        </nav>
+    </header>
+
+    <div class="hero">
+
     </div>
-    <div class="hero"></div>
-    <div class="content">
-        <div class="search-engine-optimization">
-            <img src="./assets/images/search-engine-optimization.jpg" class="float-left" />
+    <Main>
+        <div id="search-engine-optimization" class="service-section">
```

Comment:
The `main` tag has 3 `divs` and could we replace these `divs` with a semantic HTML element? It seems like the 3 `divs` are 3 main sections of the main content of the webpage. Please use a semantic HTML element here.

Reply:
didnt know this, thanks, sorted this out here 8b32038a16205c6727065cc1712ce5cd6c7a0285